tests/pure_types.go

```diff
@@ -2,9 +2,18 @@ package tests
 
 import (
 	"encoding/json"
-	"github.com/portworx/torpedo/pkg/log"
 
 	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/pkg/log"
+)
```

Comment:
can we move this file to pkg or drivers as this is not a test file?

Reply:
Good call, yes